docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,97 @@
+- `Status`: One of `Pending`, `Included`, `Verified`, or `Failed`. See [Transaction statuses section](#transaction-statuses) below.
+- `Block`: Block number containing the transaction.  
+- `Batch`: Batch number containing the transaction. 
+- `From`: The account or smart contract address sending the transaction. 
+- `Tokens Transferred`: Details on all tokens transferred by the transaction. 
+- `Contract address`:  The transaction recipient. 
+- `Input data`: Any additional data used by the transaction for verified contracts.
+- `Fee`: Fee for the tx in ETH and USD value. Click **More Details** to see info on refunds.
+- `Nonce`: Sender nonce.                         
+- `Created`: Timestamp of when the transaction was added to the block.   
 
-- Simple or asset transfers: This refers to the regular tokens transfer in the form of ether from one account to another.
+## Nature of transactions
 
-- Contract deployment transactions: Contract deployment on zkSync is quite different from Ethereum.
-  - Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` with the `data` field of the transaction equal to the contract bytecode concatenated with the constructor parameters.
-  - zkSync Era: To deploy a contract on zkSync, a user calls the `create` function of the [ContractDeployer](../system-contracts.md#contractdeployer) and provides the hash of the contract to be published, as well as the constructor arguments. The contract bytecode itself is supplied in the `factory_deps` field of the EIP712 transactions. If the contract is a factory (i.e. it can deploy other contracts), the contracts bytecodes should be included in the `factory_deps`.
-  Read more on [contract deployment](../../building-on-zksync/contracts/contract-development.md).
+There are various different ways of transacting on the Ethereum blockchain:
 
-::: tip
-zkSync Era supports Ethereum's "old" (pre-EIP-2718) transaction types, the EIP-1559 transaction type, and the EIP-712 transactions; which can be used to access zkSync-specific features such as account abstraction. Furthermore, smart contracts can only be deployed with the EIP-712 transaction type.
+- Common transactions: A simple transaction from one account to another such as transferring a token.
+- Contract deployment transactions: A transaction with no `to` address and the `data` field contains the contract code.  
+- Contract execution: A transaction that interacts with an already deployed smart contract where the `to` address is the smart contract's address.
 
-See the [EIP-712 transaction section below](#eip-712-transactions) for more information.
+### Contract deployment differences between zkSync Era and Ethereum
 
+- Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` where the `data` field contains the contract bytecode concatenated with constructor parameters.
```

Comment:
I also had a comment here that I cannot find now: above we write that contract deployment has no `to` while here we say what the `to` should be. Generally, I'm not sure what the benefit of the above is.

Reply:
removing as per above ... however, now i'm getting concerned i will get a harsh and unreasonable complaint from engineering that there's not enough words :(